q2_types/_feature_data.py

```diff
@@ -153,18 +161,20 @@ def tabular_msa_to_aligned_dna_sequences(view, data_dir):
 plugin.register_data_layout(taxonomy_data_layout)
 plugin.register_data_layout_reader('taxonomy', 1, pd.Series,
                                    taxonomy_to_pandas_series)
+plugin.register_data_layout_reader('taxonomy', 1, pd.DataFrame,
+                                   taxonomy_to_pandas_dataframe)
 plugin.register_data_layout_reader('taxonomy', 1, qiime.Metadata,
                                    taxonomy_to_qiime_metadata)
```

Comment:
Is `qiime.Metadata` redundant now? It seems with it only being an abstraction over a `df` that it is just a more restrictive (only having a `get_category` method) `df`. I mean it definitely doesn't hurt anything by having more view options, but without more `qiime.Metadata` functionality it seems like it'd only ever be viewed as a `df`? Especially since we are only giving it a `.Metadata` reader, and not a `.MetadataCategory` reader (`pd.Series` abstraction).